mne/annotations.py

```diff
@@ -783,6 +800,17 @@ def add_annotations_to_metadata(self, overwrite=False):
         -------
         self : instance of Epochs
             The modified instance (instance is also modified inplace).
+
+        Notes
+        -----
+        Annotation onsets and offsets are stored as time in seconds (not as
+        sample numbers). If the user has an Epochs object that did not support
+        Annotations before, and the Epochs were resampled, then the Annotations
+        will not show up in the correct locations. So care must be taken when
+        using this function with an Epochs object saved before Annotation
+        support was introduced.
```

Comment:
do we still need this note?  Seems like we warn pretty loudly now (in console and in docstrings) at `epochs.set_annotations()` stage,  so maybe here at the `epochs.add_annotations_to_metadata()` stage we can assume they've been done correctly?